pkg/apis/apps/v1beta1/conversion.go

```diff
@@ -82,67 +80,6 @@ func addConversionFuncs(scheme *runtime.Scheme) error {
 	return nil
 }
 
-func Convert_v1beta1_StatefulSetSpec_To_apps_StatefulSetSpec(in *StatefulSetSpec, out *apps.StatefulSetSpec, s conversion.Scope) error {
```

Comment:
I thought we needed a VolumeClaimTemplate conversion for dropping these. Is `out.VolumeClaimTemplates = *(*[]api_v1.PersistentVolumeClaim)(unsafe.Pointer(&in.VolumeClaimTemplates))` enough?